src/creates/createIssue.ts

```diff
@@ -162,6 +172,15 @@ export const createIssue = {
         dynamic: "project.id.name",
       },
     ],
-    sample: { data: { issueCreate: { success: true } } },
+    sample: {
+      data: {
+        issueCreate: {
```

Comment:
```suggestion
        issueCreate: {
          success: true,
```

We should retain `success` in the sample data, no?

Reply:
Don't believe so - on :94 we return the `issue` directly and omit the success. Has made me spot I don't think I should be returning this under an `issueCreate` though :) I'll fix that up now